### ISSUE TITLE
.exe file not generating

### ISSUE THREAD

**Issue by idk479** [ShadowMario/FNF-PsychEngine#11838]
### Describe your problem here.

so, i was making a fixed build of Vs dave definitive edition and there was NOTHING wrong with the code, compiling it, etc.
But the .exe file just doesn't generate. So i can't actually test the game myself. I think it's probably my pc's problem or the haxe version the source code uses is old. But i still need clarification.

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows x64

### Did you edit anything in this build? If so, mention or summarize your changes.

Edited and added some new stuff in source.

**Comment by PDL2CGamejolt** [comment]
Wait,,,, you were making- A FIXED BUILD?

**Comment by idk479** [comment]
yeah, but it's not going public unless i got perms from whatsdown
